src/io/read_tree_consistentrees_ascii.c

```diff
@@ -136,10 +141,28 @@ int setup_forests_io_ctrees(struct forest_info *forests_info, const int ThisTask
     XASSERT(ctr->tree_fd != NULL, MALLOC_FAILURE, "Error: Could not allocate memory to store the file descriptor per tree\n"
```

Comment:
Throughout this function, can the asserts be converted to returns?

Reply:
So many ASSERTS!